front/types/assistant/conversation.ts

```diff
@@ -126,6 +126,21 @@ export type ConversationType = {
   title: string | null;
   visibility: ConversationVisibility;
   content: (UserMessageType[] | AgentMessageType[])[];
+  participants: {
+    users: UserParticipant[];
+    agents: AgentParticipant[];
+  };
+};
+
+export type UserParticipant = {
+  userName: string;
```

Comment:
to be consistent with UserMessageContext type this should be `username` and `fullName`

Reply:
I added a commit!